config/config.go

```diff
@@ -1193,7 +1193,9 @@ func (p *PackageConfig) LocalDataParentDirectory(packagesDir string) string {
 
 // SanitizedName returns the package name for the symlink/filepath of the package on the system.
 func (p *PackageConfig) SanitizedName() string {
-	return fmt.Sprintf("%s-%s", strings.ReplaceAll(p.Package, string(os.PathSeparator), "-"), p.sanitizedVersion())
+	// p.Package is set by the PackageServiceClient as "{org_id}/{package_name}"
+	// see https://github.com/viamrobotics/app/blob/main/packages/packages.go#L1257
+	return fmt.Sprintf("%s-%s", strings.ReplaceAll(p.Package, "/", "-"), p.sanitizedVersion())
```

Comment:
If somebody one day changed `FormatPackageID` [from using /](https://github.com/viamrobotics/app/blob/4d95cecd31a9e384fd5a970a329ee5c672473abe/packages/packages.go#L1372) to using `os.PathSeparator`, is that going to introduce a problem again?

Reply:
well the app packageservice client runs on app, so i think os.PathSeparator would always return `/` as we run on unix boxes. I guess if formatpackageid was made to be context aware of the package platform, then yes it would introduce an issue.

Reply:
Ah, got it. Agreed that changing that to `os.PathSeparator` would just be wrong and chaotic.Thanks for the context!

Reply:
Totally outside of the scope of this change or the issue I'm trying to address but I think PackageID should be a hash or UUID that we don't have to sanitize. Definitely a much larger change and probably there is context that i'm unaware of on why we did it this way.

Reply:
Yeah -- and the fact that that app generated string is turned into a path thing is a smell.